lib/vtls/rustls.c

```diff
@@ -1008,6 +1008,11 @@ init_config_builder_ech(struct Curl_easy *data,
     goto cleanup;
   }
 cleanup:
+  /* if we base64 decoded, we can free now */
+  if(data->set.tls_ech & CURLECH_CLA_CFG
+       && data->set.str[STRING_ECH_CONFIG]) {
+      free(ech_config);
```

Comment:
```suggestion
  if((data->set.tls_ech & CURLECH_CLA_CFG) &&
     data->set.str[STRING_ECH_CONFIG]) {
    free(ech_config);
```
indentation/formatting nits

Reply:
actually those are short enough for one line